### ISSUE TITLE
Hilarious, but admittedly irritating, rat dupe glitch

### ISSUE THREAD

**Issue by rustedgoldmc** [Ladysnake/Rats-Mischief#25]
I love the rats and I've been playing around with them in Creative. I particularly enjoy getting a metric ton of rats and having them absolutely massacre mobs. However, this did lead me to discover something that is a tad odd. When I already have several sacks of rats in my inventory and just a few left to pick up, and I shift+right click to get them all at once, the empty space in the bag will fill up with rats whose ID is "ratsmischief:rat" (or something very similar to that, I don't have the game open) and when I empty the bag there are the couple of rats who I own and then literal dozens of brand new rats. I know they were brand new, as well, because one was a gold rat and I hadn't had any of those. It gets quite irritating because from that point forward the rat sacks keep filling themself with brand new not-mine rats when I try to fill them and there aren't exactly 20 or more around (I have only tested this with the purpur sack because I always use a metric ton of rats) and sometimes it leads to rats that do belong to me getting left out while the sack fills completely with new rats. Anyway, it's quite funny and could probably be exploited for Infinite Rats, but it should probably be fixed. Also, the rat mask doesn't have a texture - I put it on and it makes a magenta cube around my head both in the inventory and in f5 mode. I would be fully willing to give more specific information that I didn't think of. Thank you :)

**Comment by doctor4t** [comment]
I can't seem to be able to reproduce that bug. I see that you have an etheric gem from Respawnable Pets. Did you do anything with it?

**Comment by doctor4t** [comment]
The mask of rat also seems fine, maybe you could provide a list of what mods you are using with Mischief?

**Comment by rustedgoldmc** [comment]
Yes, I had used the gem on all the rats. And I've included the .zip file of my mods folder in with the comment, but I'm not sure if it worked.

**Comment by rustedgoldmc** [comment]
It did not work, and I couldn't upload the .zip file when I tried again, so here are screenshots of my mods folder.
![image](https://user-images.githubusercontent.com/83943239/119269662-6786cd00-bbbe-11eb-8d16-e94a42c80b79.png)
![image](https://user-images.githubusercontent.com/83943239/119269688-8e450380-bbbe-11eb-8ebd-7a2e25734a0e.png)
![image](https://user-images.githubusercontent.com/83943239/119269695-9ac95c00-bbbe-11eb-844a-91319200840f.png)
![image](https://user-images.githubusercontent.com/83943239/119269701-a4eb5a80-bbbe-11eb-9f60-e681ef4942bd.png)
I have a lot of mods, I know. I did test them all and they all work together, the two things I mentioned are the only issues I've found in the entire folder. Neither of the issues are serious enough for me to remove the rat mod, either.
